src/Stryker.Core/Stryker.Core/ProjectComponents/IReadOnlyInputComponent.cs

```diff
@@ -13,8 +13,6 @@ public interface IReadOnlyInputComponent
         IEnumerable<IReadOnlyMutant> ReadOnlyMutants { get; }
         IEnumerable<IReadOnlyMutant> TotalMutants { get; }
         IEnumerable<IReadOnlyMutant> DetectedMutants { get; }
-        IReadOnlyCollection<FileLeaf> ExcludedFiles { get; }
-        bool IsExcluded { get; }
```

Comment:
We do not want to lose this information as we will want to explicitly display to users in reporting that they excluded a given file.

Reply:
Nevermind, I forgot you added that to the Mutant.